.github/workflows/gnulinux.yml

```diff
@@ -0,0 +1,280 @@
+          sudo apt-get install -y \
+            cmake \
+            build-essential \
+            automake \
+            gtk-doc-tools \
+            rpm \
+            rename \
+            sshpass \
+            clang-6.0 \
+            g++-8 \
+            gcc-multilib \
+            g++-multilib \
+            libgtest-dev \
+            libc++-dev \
+            libogg-dev \
+            libvorbis-dev \
+            libopenal-dev \
+            libboost-all-dev \
+            libsdl2-dev \
+            libsdl2-image-dev \
+            libfreetype6-dev \
+            libharfbuzz-dev \
+            libfribidi-dev \
+            libglib2.0-dev \
+            libraqm-dev \
+            libglew-dev \
+            libglbinding-dev \
+            libcurl4-openssl-dev \
+            libglm-dev \
+            zlib1g-dev \
+            lcov
+
+      - name: Install 32-bit dependencies
+        if: ${{ matrix.arch == 32 }}
+        env:
+          DOWNGRADE_PCRE: ${{ matrix.os == 'ubuntu-latest' && '1' || '' }}
+        run: |
+          sudo dpkg --add-architecture i386
+          sudo apt-get update
+
+          if [ "$DOWNGRADE_PCRE" ]; then
+            # Github is adding a lot of unnecessary deb.sury.org
+            # packages into ubuntu-latest, this causes
+            # libharfbuzz-dev:i386 to fail due to issues related to
+            # libpcre2-8-0 from deb.sury.org. Remove all that and
+            # downgrade to official versions.
+            sudo apt-get remove --yes php.* libzip4:amd64
+            sudo apt-get install --yes --allow-downgrades \
+              libpcre2-16-0:amd64=10.34-7 \
+              libpcre2-32-0:amd64=10.34-7 \
+              libpcre2-8-0:amd64=10.34-7 \
+              libpcre2-dev:amd64=10.34-7 \
+              libpcre2-posix2:amd64=10.34-7
+          fi
+
+          sudo apt-get install -y \
+            cmake \
+            build-essential \
+            automake \
+            gtk-doc-tools \
+            rpm \
+            rename \
+            sshpass \
+            gcc-multilib \
+            g++-multilib \
+            libgtest-dev:i386 \
+            libogg-dev:i386 \
+            libvorbis-dev:i386 \
+            libopenal-dev:i386 \
+            libboost-date-time-dev:i386 \
+            libboost-filesystem-dev:i386 \
+            libboost-locale-dev:i386 \
+            libsdl2-dev:i386 \
+            libsdl2-image-dev:i386 \
+            libfreetype6-dev:i386 \
+            libcurl4-openssl-dev:i386 \
+            libharfbuzz-dev:i386 \
+            libfribidi-dev:i386 \
+            libglm-dev \
+            zlib1g-dev
+
+          # Nethier GLEW nor glbinding exist in 32-bit for Ubuntu 20.04, so snatch the debs from 16.04 instead
+          wget archive.ubuntu.com/ubuntu/pool/main/g/glew/libglew1.13_1.13.0-2_i386.deb && sudo dpkg -i libglew1.13_1.13.0-2_i386.deb
+          wget archive.ubuntu.com/ubuntu/pool/main/g/glew/libglew-dev_1.13.0-2_i386.deb && sudo dpkg -i libglew-dev_1.13.0-2_i386.deb
```

Comment:
lol

Reply:
It has been the case since 32-bit Linux builds existed :smile: